src/components/TextArea.jsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Grid, TextField } from '@mui/material';
+import TextareaAutosize from '@mui/material/TextareaAutosize';
+
+export default function TextArea() {
+  return (
+      <Grid container spacing={1}>
+        <Grid item xs={12} md={12}>
```

Comment:
Please change the layout of this grid items to be a row and not columns, in other words change the size from 12 to what ever fits.